src/System.IO.Pipes/tests/AnonymousPipesThrowsTests.cs

```diff
@@ -1,17 +1,15 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.IO.Pipes;
-using System.Threading.Tasks;
-using Xunit;
-
 using Microsoft.Win32.SafeHandles;
+using Xunit;
 
 namespace System.IO.Pipes.Tests
 {
     public class AnonymousPipesThrowsTests
     {
+        private static void NotReachable(object obj) { Assert.True(false, "This should not be reached."); }
```

Comment:
Perhaps XUnit needs a call like Assert.ThrowsAsyncImmediate()  To ensure the throw happens before the task is created (I assume that is the purpose of NotReachable())